android/src/main/java/com/theoplayer/PlayerEventEmitter.kt

```diff
@@ -320,7 +320,7 @@ class PlayerEventEmitter internal constructor(
   private fun onPause() {
     val player = playerView.player
     // Do not forward the pause event in case the content player is paused because the ad player starts.
-    if (player != null && !playerView.adsApi.isPlaying) {
+    if (player != null && !(playerView.adsApi.isPlaying && !player.isPaused)) {
```

Comment:
```suggestion
    if (!playerView.adsApi.isPlaying && player.isPaused) {
```

Reply:
It should be:
`if (!playerView.adsApi.isPlaying || player.isPaused)`
right?